spec/indexers/hyrax/valkyrie_collection_indexer_spec.rb

```diff
@@ -3,9 +3,8 @@
 require 'hyrax/specs/shared_specs'
 
 RSpec.describe Hyrax::ValkyrieCollectionIndexer do
-  subject(:indexer) { described_class.new(resource: resource) }
+  let(:resource) { FactoryBot.valkyrie_create(:hyrax_collection) }
   let(:indexer_class) { described_class }
 
-  it_behaves_like 'a Hyrax::Resource indexer'
```

Comment:
All shared specs representing the minimal required parts of a collection are now defined and tested in the shared spec 'a Collection indexer'.